node/node.go

```diff
@@ -1444,7 +1454,27 @@ func saveGenesisDoc(db dbm.DB, genDoc *types.GenesisDoc) error {
 	if err != nil {
 		return fmt.Errorf("failed to save genesis doc due to marshaling error: %w", err)
 	}
-	if err := db.SetSync(genesisDocKey, b); err != nil {
+
+	blockSize := 100000000 // 100mb
```

Comment:
nit: Please help me understand. Why do we write with every 100M bytes?

Reply:
It doesn't matter what value it is. When I set the value to 1GB and tried to save 2GB in the unit test, a ci error occurred due to a memory issue. For this reason, I changed it to a value that satisfies this for the unit test. Even in this case, 256 * 100MB = 25.6GB, so most genesis.json files should have no problems.

Reply:
I got it! Thanks. Could you write down its spec in the PR description?